File: Codigos_por_competidor/Alejandra_Carvajal/boyOrGirl.py
# Autor: Alejandra Carvajal
# Problema: 236A - Boy or Girl
# Juez online: Codeforces
# Veredicto: Runtime error on test 1
# URL: https://codeforces.com/problemset/problem/236/A
t = int(input())
for i in range(1,t+1):
    user = input()
    
    u = set(user)
    c = len(u)
    
    if c % 2 == 0:
        print("CHAT WITH HER!")
    else:
        print("IGNORE HIM!")

File: Resueltos_por_competidor/Gina_Sánchez/BoyOrGirl.java
/*
 * Autor: Gina Sofia Sánchez Estupiñan
 * Problema: Boy or Girl
 * Juez online: Codeforces
 * Veredicto: Accepted
 * URL: https://codeforces.com/problemset/problem/236/A
 */

import java.util.Scanner;

public class BoyOrGirl {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		while (sc.hasNext()) {
			String p = "";
			String pf = "";
			p = sc.nextLine();

			for (int i = 0; i < p.length(); i++) {
				char c = p.charAt(i);
				String pc = "";
				pc += c;
				if (!pf.contains(pc)) {
					pf += pc;
				}
			}

			if (pf.length() % 2 == 0) {
				System.out.println("CHAT WITH HER!");
			} else {
				System.out.println("IGNORE HIM!");
			}
		}
	}

}
